route/wraith_island/necromancers_lair.py

```diff
@@ -404,7 +404,7 @@ def __init__(self: Self) -> None:
                 SeqInteract("Graplou"),
                 SeqSkipUntilIdle("Graplou"),
                 SeqCombatAndMove(
-                    name="Graplouing around",
+                    name="Graplou movement",
```

Comment:
Aww, but I like using Graplouing as a verb :P

Reply:
😁
I'll forward your complaint to my spell checker :P